wurst/util/EffectUtils.wurst

```diff
@@ -0,0 +1,135 @@
+package EffectUtils
+import NoWurst
+import Effect
+import Vectors
+import Colors
+import Player
+
+public class EffectEx
```

Comment:
I'm not a fan of the name 'Ex' name and the package name, and, this should be another PR.
Please keep PRs as short as possible and encapsulated, for faster review and merge.

Reply:
Sure, I'll keep that in mind and I'll remove this package in this PR.